src/common/options.cc

```diff
@@ -7915,7 +7915,7 @@ static std::vector<Option> get_immutable_object_cache_options() {
     .set_description("immutable object cache client dedicated thread number"),
 
     Option("immutable_object_cache_watermark", Option::TYPE_FLOAT, Option::LEVEL_ADVANCED)
```

Comment:
Nit: since you are fixing this (by inverting the value), perhaps add a small note to the `PendingReleaseNotes` file stating that the `immutable_object_cache_watermark` has been updated to property reflect the upper cache utilization before space is reclaimed.